recipe/meta.yaml

```diff
@@ -36,10 +36,25 @@ requirements:
     - traitlets
 
 test:
+  requires:
+    - nose
+    - setuptools
+    - wheel
+    - nose
+    - coverage
+    - nodejs
+    - phantomjs
```

Comment:
> Running the tests in the packager should only test "was the package built correctly,"

I guess some clarification on the documentation would be appropriate, then, as the home page encourages tests to be run... and for a number of packages, this might be the first time they are run on non-developerly situations, despite their limitations.

> Mainly because these tests (and testing tools) are of insufficient quality

Welp, we gotta make it better, then. For example, going to phantomjs 2 (which this does) is already working pretty well, and requires fewer hacks. It's still based on qtwebkit. Nightmarejs would be an option as well, which uses electron (chromium). The native browsers approach (with selenium) would seem to be the endgame, but does require a large number of dependencies (jdk). 

As we are trying to encourage more robust tests across the community, I'd really hope we get some "use this and you'll be good" assurance from the flagship project... Over on https://github.com/Anaconda-Server/anaconda-nb-extensions/, ideally as each extension is added, we will need to run:
- the original notebook tests, to ensure no regressions have been introduced
  - or if it changes a core feature, be able to denote this
- new integration tests for the extension
- the integration tests for the other extensions

To achieve this, we'll need a good deal of commonality in the test approach, or we'll be installing testing tools for hours.